src/commands/status/core.js

```diff
@@ -0,0 +1,114 @@
+const { table } = require('table');
+const fetch = require('node-fetch');
+
+const BaseCommand = require('../../oclif/command/BaseCommand');
+const CoreService = require('../../core/CoreService');
+
+const ContainerIsNotPresentError = require('../../docker/errors/ContainerIsNotPresentError');
+
+class CoreStatusCommand extends BaseCommand {
+  /**
+   * @param {Object} args
+   * @param {Object} flags
+   * @param {DockerCompose} dockerCompose
+   * @param {CoreService} coreService
+   * @param {Config} config
+   * @return {Promise<void>}
+   */
+  async runWithDependencies(
+    args,
+    flags,
+    dockerCompose,
+    createRpcClient,
+    config,
+  ) {
+    const rows = [];
+
+    const coreService = new CoreService(
+      createRpcClient(
+        {
+          port: config.get('core.rpc.port'),
+          user: config.get('core.rpc.user'),
+          pass: config.get('core.rpc.password'),
+        },
+      ),
+      dockerCompose.docker.getContainer('core'),
+    );
+
+    let insightUrl;
+    switch (config.options.network) {
+      case 'evonet': insightUrl = 'http://insight.evonet.networks.dash.org:3001/insight-api'; break;
+      case 'testnet': insightUrl = 'https://testnet-insight.dashevo.org/insight-api'; break;
+      case 'mainnet': insightUrl = 'https://insight.dash.org/insight-api'; break;
+      default: insightUrl = 'https://insight.dash.org/insight-api';
+    }
+
+    // Collect data
+    const blockchainInfo = (await coreService.getRpcClient().getBlockchainInfo()).result;
+    const networkInfo = (await coreService.getRpcClient().getNetworkInfo()).result;
+    const mnsyncStatus = (await coreService.getRpcClient().mnsync('status')).result;
+    const peerInfo = (await coreService.getRpcClient().getPeerInfo()).result;
+    const latestVersion = JSON.parse(await fetch('https://api.github.com/repos/dashpay/dash/releases/latest').then((res) => res.text()));
```

Comment:
You can use object destruction to set a specific property to a variable.

Reply:
Usually, it’s better to put execution rest to variable and pass it to the next function. It’s much more readable than oneliners. Also, I would recommend to do not combine the await and promise-like approach. This is considered smelling according to best practices.